pkg/webui/console/views/organization-overview/organization-overview.js

```diff
@@ -58,34 +74,44 @@ class Overview extends React.Component {
     ]
 
     return (
-      <div>
-        <div className={style.title}>
-          <h2>{name || ids.organization_id}</h2>
-          {description && <span className={style.description}>{description}</span>}
-        </div>
-        <DataSheet data={sheetData} />
-      </div>
-    )
-  }
-
-  render() {
-    const { organization } = this.props
-    const orgId = getOrganizationId(organization)
-
-    return (
-      <Container>
-        <IntlHelmet title={sharedMessages.overview} />
-        <Row>
-          <Col sm={12} lg={6}>
-            {this.organizationInfo}
-          </Col>
-          <Col sm={12} lg={6}>
-            <div className={style.latestEvents}>
+      <React.Fragment>
+        <EntityTitleSection
+          entityId={orgId}
+          entityTitle={name}
```

Comment:
this should be `organization.name`

Reply:
Why? `name` already is from the organization. See L54.

Reply:
ah my bad. the issue is that you provided `entityTitle` here instead of `entityName`

Reply:
Right, good catch!